tests/python/unittest/test_ndarray.py

```diff
@@ -862,7 +862,7 @@ def test_iter():
     for i in range(x.size):
         assert same(y[i].asnumpy(), x[i].asnumpy())
 
-@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/8049")
+@with_seed()
 def test_cached():
```

Comment:
Please put the link to the tracking issue in a comment like:
```Python
# @<your-id>: Tracked at https://github.com/apache/incubator-mxnet/issues/8049.
```

Reply:
@haojin2  Re-submitted with this change. Thanks.